be/src/exec/parquet_writer.cpp

```diff
@@ -0,0 +1,197 @@
+
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/parquet_writer.h"
+
+#include <fmt/format.h>
+
+#include "runtime/exec_env.h"
+#include "util/priority_thread_pool.hpp"
+#include "util/uid_util.h"
+
+namespace starrocks {
+
+ParquetWriterWrap::ParquetWriterWrap(const TableInfo& tableInfo, const PartitionInfo& partitionInfo,
```

Comment:
What's difference between `ParquetWriterWrap` and `ParquetWriter`?  Why to emphasize `wrap`？

Reply:
How about RollingAsyncParquetWriter?